asaplib/plot/plot_colors.py

```diff
@@ -104,7 +104,7 @@ def set_color_function(fcolor=None, fxyz=None, colorscol=0, n_samples=0, peratom
     colorlabel = 'use ' + fcolor + ' for coloring the data points'
 
     if peratom:
-        print(np.shape(plotcolor_atomic))
+        #print(np.shape(plotcolor_atomic))
```

Comment:
Would a switch for printing the shape of `plotcolor_atomic` be worth it here? Will users be expected to debug this by printing the shape?